### PR TITLE
📝 Add Apus to the list of projects using BigBone

### PR DESCRIPTION
**Pull request by McPringle**
# Description

I added the open source project *Apus* to the list of projects using BigBone.

# Type of Change

- Documentation

# Breaking Changes

No breaking changes.

# How Has This Been Tested?

No tests (just documentation).

# Mandatory Checklist

- [ ] ~~I ran `gradle check` and there were no errors reported~~
- [x] I have performed a self-review of my code
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [ ] ~~All tests pass locally with my changes~~
- [ ] ~~I have added KDoc documentation to all public methods~~

### CODE REVIEW

File: README.md
Comment (PattaFeuFeu) [inline README.md:249]:
```suggestion
  with Mastodon posts containing a configurable hashtag. Its purpose is to be displayed at the event on projector and TVs
```

**Review by PattaFeuFeu — COMMENTED** [review]
I would suggest changing “beamer” to “projector” as “beamer” isn’t an English word used for that device, but instead a pseudo-Anglicism used in German languages 😊

**Review by PattaFeuFeu — APPROVED** [review]
Fine by me but waiting on @andregasser’s approval for a merge

**Review by andregasser — APPROVED** [review]
Thanks for adding your project to our list 😊. All good!

**Pull request merged**
